src/mate/academy/homework7/directory/annotation/InjectorHuman.java

```diff
@@ -0,0 +1,36 @@
+package mate.academy.homework7.directory.annotation;
+
+import mate.academy.homework7.directory.dao.FileHumanDao;
+import mate.academy.homework7.directory.dao.HumanDao;
+import mate.academy.homework7.directory.dao.InMemoryHumanDao;
+import mate.academy.homework7.directory.factory.HumanDaoFactory;
+import mate.academy.homework7.directory.handler.ConsoleHandlerHuman;
+
+import java.lang.reflect.Field;
+
+public class InjectorHuman {
+    public static void injectDependency() throws IllegalAccessException {
+
+        Class consoleHumanHandlerClass = ConsoleHandlerHuman.class;
```

Comment:
create field with list of classes and make this method to be generic, not bounded to 3 classes, it should work for any class you pass

Reply:
@Siril-O Why you don`t check my home work again? I have 4 red HW.... Please, check it!